web/templates/message_body.hbs

```diff
@@ -55,8 +73,18 @@
     {{/unless}}
 {{/unless}}
 
+{{#if editing_in_left_col}}
+    <div id="editing_notifications" class="edit_notifications">
+        {{#if is_typing}}
+            {{> editing_notifications typing=true}}
+        {{/if}}
+    </div>
+{{/if}}
+
 {{#if edited_in_left_col}}
+{{#unless is_typing}}
 {{> edited_notice}}
+{{/unless}}
```

Comment:
These changes are super messy. We should do a preparatory PR that refactors to have the template render with unconditionally set variables `edited_notices_in_left_col` and then just have an `is_typing` if/else block added inside that container.

You can do that preparatory refactoring PR and then come back to this -- we should be able to merge that change on top of `main`.

Reply:
Updated!